src/Video/Renderer.hpp

```diff
@@ -30,6 +30,17 @@ namespace Video {
             /// Destructor.
             VIDEO_API ~Renderer();
 
+            ////Prepare for shadow renderin
+            /**
+            * @param lightView The lights view matrix
+            * @param lightProjection the Lights projection matrix
+            * @param shadowId the texture ID for the shadowMap
+            * @param shadowWidth size x of shadowMap
+            * @param shadowHeight size y of shadowMap
+            * @param depthFbo framebuffer object for shadowMap
+            */
+            VIDEO_API void PrepareShadowRendering(const glm::mat4 lightView, glm::mat4 lightProjection, int shadowId,int shadowWidth, int shadowHeight, int dephtFbo);
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') comma should be followed by whitespace [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT009)